app/views/datasets/_timeseries_data.html.erb

```diff
@@ -1,33 +1,21 @@
-<section>
-  <div class="grid-row">
-    <div class="column-full">
-      <section class="dgu-datalinks">
-        <!-- accordions for time series datasets -->
-        <h2 class="heading-medium">Data links
-          <span class="showHide-open-all">Open all</span>
-        </h2>
-        <div class="breaker"></div>
-        <ul>
-          <% datafiles.each do |year, datafiles| %>
-            <li class="showHide">
-              <div class="year-expand showHide-control">
-                <h3 class="heading-small">
-                  <button class="button secondary">
-                    <%= year %>
-                  </button>
-                </h3>
-                <div>
-                  <button class="expand button secondary">+</button>
-                </div>
-              </div>
-              <div class="year-datasets showHide-content" style="display:none">
-                <%= render partial: 'datafile_table', locals: { datafiles: datafiles } %>
-              </div>
-          <% end %>
-          </li>
-        </ul>
-        <input class="button dgu-datalinks__download" type="submit" value="Download all">
-      </section>
+<ul>
+  <% datafiles.group_by(&:start_year).each do |year, datafiles| %>
+    <li class="showHide">
+    <div class="year-expand showHide-control">
+      <h3 class="heading-small">
+        <button class="button secondary">
+          <%= year %>
+        </button>
+      </h3>
+      <div>
+        <button class="expand button secondary">+</button>
+      </div>
     </div>
-  </div>
-</section>
+    <div class="year-datasets showHide-content" style="display:none">
```

Comment:
I don't think that `year-datasets` is used. Feel free to remove, if it doesn't break things.